### PR TITLE
[CBRD-25147] Add new test case for SQL syntax to add users as members of the group

### PR DESCRIPTION
**Pull request by zionyun**
refer to http://jira.cubrid.org/browse/CBRD-25147

Error msg
168 Member not found.
493 Syntax: %1$s %2$s
142 Adding member causes the user hierarchy to become a cyclic graph.
167 DBA, members of DBA group and class owner can perform the operation.

### CODE REVIEW

**Review by ssihil — APPROVED** [review]

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+-- Verify the syntax: ALTER USER user_name PASSWORD password;
+ALTER USER test_user1 PASSWORD '1234';
+CALL login ('test_user1', '1234') ON CLASS db_user;
+
+CALL login ('dba', '') ON CLASS db_user;
+-- Verify the syntax: ALTER USER user_name COMMENT 'comment_string';
+ALTER USER test_user1 COMMENT '1) comment';
+SELECT name, comment FROM db_user WHERE name = 'TEST_USER1';
+
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql:20]: add
ALTER USER test_user1 COMMENT '1) ''comment';
SELECT name, comment FROM db_user WHERE name = 'TEST_USER1';

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+-- Verify the syntax: ALTER USER user_name ADD MEMBERS user_name;
+ALTER USER test_user1 ADD MEMBERS test_user2;
+SELECT u.name, x.name as groups FROM db_user AS u, TABLE(u.direct_groups) AS g(x);
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql]: modify

SELECT u.name, x.name as groups, u.comment FROM db_user AS u, TABLE(u.direct_groups) AS g(x) order by u.name;

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+-- Verify the syntax: ALTER USER user_name DROP MEMBERS user_name;
+ALTER USER test_user1 DROP MEMBERS test_user2;
+SELECT u.name, x.name as groups FROM db_user AS u, TABLE(u.direct_groups) AS g(x);
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql]: modify

SELECT u.name, x.name as groups, u.comment FROM db_user AS u, TABLE(u.direct_groups) AS g(x) order by u.name;

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+-- Attempt to use incorrect syntax: should raise an error:-493
+CALL login ('dba', '') ON CLASS db_user;
+ALTER USER test_user1 PASSWORD '1234' ADD MEMBERS test_user2;
+ALTER USER test_user1 PASSWORD '1234' DROP MEMBERS test_user2;
+ALTER USER test_user1 ADD MEMBERS test_user2 DROP MEMBERS test_user2;
+
+-- Verify the syntax: ALTER USER user_name ADD MEMBERS user_name COMMENT ‘comment_string’;
+ALTER USER test_user1 ADD MEMBERS test_user2 COMMENT '3) add members, comment';
+SELECT u.name, x.name as groups, u.comment FROM db_user AS u, TABLE(u.direct_groups) AS g(x);
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql]: Add an "order by" clause.

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+-- Verify the syntax: ALTER USER user_name DROP MEMEBERS user_name COMMENT ‘comment_string’;
+ALTER USER test_user1 DROP MEMBERS test_user2 COMMENT '4) drop members, comment';
+SELECT u.name, x.name as groups, u.comment FROM db_user AS u, TABLE(u.direct_groups) AS g(x);
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_1.sql]: Add an "order by" clause.

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_2.sql [hunk 6]
```diff
@@ -0,0 +1,36 @@
+-- This scenario varifies CBRD-25147 issue.
+-- Add SQL syntax to add users as members of the group.
+
+-- 2. If DBA and DBA group members add general users, verify that the general users have DBA privileges
+
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_2.sql:5]: Add an “order by” clause to the SELECT query statement below.

File: sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_3.sql [hunk 7]
```diff
@@ -0,0 +1,46 @@
+-- This scenario varifies CBRD-25147 issue.
+-- Add SQL syntax to add users as members of the group.
+
+-- 3. General users can only add or drop members in their own groups
+
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25147/cases/cbrd_25147_3.sql:5]: Add an “order by” clause to the SELECT query statement below.

**Review by jongmin-won — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]

**Review by tw-kang — APPROVED** [review]

**Review by bagus-kim — APPROVED** [review]

**Pull request merged**